metadata-models/src/main/pegasus/com/linkedin/policy/DataHubPolicyInfo.pdl

```diff
@@ -42,6 +48,12 @@ record DataHubPolicyInfo {
   /**
    * The privileges that the policy grants.
    */
+  @Searchable = {
+    "/*": {
+      "fieldType": "KEYWORD",
+      "addToFilters": true
```

Comment:
What does`addToFilters` allow us to do?

Reply:
See https://datahubproject.io/docs/next/metadata-modeling/extending-the-metadata-model/#searchable  - This is to allow the array values to be used as filters from what I understand.